xclim/core/indicator.py

```diff
@@ -318,11 +277,12 @@ class Indicator(IndicatorRegistrar):
        "kind" refers to the constants of :py:class:`xclim.core.utils.InputKind`.
     """
 
-    cf_attrs: Sequence[Mapping[str, Any]]
+    output: Sequence[Mapping[str, Any]] = None
     """A list of metadata information for each output of the indicator.
 
        It minimally contains a "var_name" entry, and may contain : "standard_name", "long_name",
-       "units", "cell_methods", "description" and "comment".
+       "units", "cell_methods", "description" and "comment" on official xclim indicators. Other
+       fields could also be present if the indicator was created from outside xclim.
```

Comment:
We can discuss the name of this attribute.  I want it to be the same as the section in the yaml.

`cf_attrs` is false because we can now accept non-cf attrs (current example: `proposed_standard_name` from clix-meta indices).

Reply:
- nc_attrs 
- metadata
- var_attrs

Reply:
I'd go with "metadata" or "attrs".

What does @Zeitsperre think?  This is the object with the unformatted attributes of the indicator's output. It is a list of dictionaries. This is also the name that will be used in the yaml.

Reply:
Thinking of it `metadata` doesn't carry the distinction between output variable attributes and general metadata. (Like `title` vs `long_name`, `abstract` vs `description`) And neither does `attrs`.

Reply:
If I understand correctly, I would maybe go with `inherited_attrs`?

Reply:
I think you are technically correct, the attrs are inherited from the indice. However, I feel that this notion will be alien for the user. 
The branch currently uses "var_attrs", which convey the meaning I want it to convey, but I am open for more discussion, it's not my favorite.

Reply:
Why again are we ditching cf_attrs ? Because one new attribute is not strictly_required by the CF convention ? I don't feel this is a strong enough reason to break things.

Reply:
well. I realize that now. 
At first it's because I didn't want `cf_attrs` as the section name in the yaml, but now  `var_attrs`  is not drastically different, so why change? I am also ok with accepting non-cf attrs in `cf_attrs`.
Sorry for the noise!

Reply:
Pas de trouble, content quand ça arrive ici aux autres !